executor/ddl_test.go

```diff
@@ -1105,3 +1105,13 @@ func (s *testRecoverTable) TestRenameTable(c *C) {
 	tk.MustExec("drop database rename1")
 	tk.MustExec("drop database rename2")
 }
+
+func (s *testRecoverTable) TestDropTableIfExists(c *C) {
```

Comment:
Please move this test to: https://github.com/pingcap/tidb/blob/1aa0a7d6b304fb3efb1a2fe5689db07b21168d5d/executor/ddl_test.go#L77

, add tests for `VIEW` here:
https://github.com/pingcap/tidb/blob/1aa0a7d6b304fb3efb1a2fe5689db07b21168d5d/executor/ddl_test.go#L178

and add tests for `SEQUENCE` here:
https://github.com/pingcap/tidb/blob/1aa0a7d6b304fb3efb1a2fe5689db07b21168d5d/ddl/sequence_test.go#L90

BTW, as you may noticed, the testing codes for `DDL` is a mess, we have a plan to re-organize them.

Reply:
All fixed.